observability/WFLY-20296-add-support-for-more-opentelemetry-config-options.adoc

```diff
@@ -0,0 +1,154 @@
+||Compression|`otel.exporter.otlp. compression`|The compression type to use on OTLP trace, metric, and log requests.|`compression`|null|Only `gzip` supported
+|Traces|Enabled|`otel.traces.exporter`|Are traces enabled?|`traces-enabled`|true|If true, the value is set to `otlp`. If not, the value is set to `none`.
+|Metrics|Enabled|`otel.metrics.exporter`|Are metrics enabled?|`metrics-enabled`|true|If true, the value is set to `otlp`. If not, the value is set to `none`.
+||Export interval|`otel.metrics.export.interval`|The interval, in milliseconds, between the start of two export attempts.|`metrics-export-interval`|60000|
+||Exemplar filter|`otel.metrics.exemplar.filter`|The filter for exemplar sampling.|`exemplar-filter`|`trace_based`|Options are `always_off`, `always_on` or `trace_based`.
+||Metrics cardinality limit|`otel.experimental.metrics. cardinality.limit`|If set, configure cardinality limit; the maximum number of distinct points per metric.|cardinality-limit|2000|
+||Temporality preference|`otel.exporter.otlp.metrics. temporality.preference`|The preferred output aggregation temporality.|`temporality`|`cumulative`|Options include `delta`, `lowmemory`, and `cumulative`.
+||Histogram aggregation|`otel.exporter.otlp. metrics.default. histogram.aggregation`|The preferred default histogram aggregation.|`histogram-aggregation`|`explicit_bucket_histogram`|Options include `base2_exponential_bucket_histogram` and `explicit_bucket_histogram`
+|Logs|Enabled|`otel.logs.exporter`|Are logs enabled?|`logs-enabled`|true|If true, the value is set to `otlp`. If not, the value is set to `none`.
+||Schedule delay|`otel.blrp.schedule.delay`|The interval, in milliseconds, between two consecutive exports.|`logs-export-interval`|1000|
+|===
+* Changes to the management model should be split out based on signal types
+** For ease of use by users, the structure of the updated XML should reflect this grouping of signal-specific attributes via `setAttributeGroup(String)`
+** The shape of the management model should be flat, using a list of attributes, rather than nested resources
+*** Each attribute will be declared on the existing `OpenTelemetrySubsystemRegistrar`. No new child resources should be created.
+*** Attribute groups will be used to group properties by signal type to make the XML file and admin console more user-friendly
+** Shared properties (e.g., the OpenTelemetry Collector endpoint) should be exposed on the root element.
+*** Signal-specific overrides need not be supported
+* All new properties added will have, as defaults, values described in the relevant OpenTelemetry documentation (see the official docs https://opentelemetry.io/docs/languages/java/configuration/[here])
+* Support should not depend on, e.g., MicroProfile Telemetry or MicroProfile Config
+* Any configured properties should still be able to be overridden via the existing MicroProfile support (this is the current default behavior)
```

Comment:
The textual description of any such attribute must explicitly declare that if MicroProfile Telemetry is used the config property 'foo' takes precedence.

I hope if MP Tel is not used that system properties are not preferred to what is set via the management API. If that's not the case let's have a zulip or dev list discussion.